### ISSUE TITLE
Local provider provider URL not configurable / doesn't use standard Ollama URL/port

### ISSUE THREAD

**Issue by sammcj** [ad-si/cai#4]
It seems odd that cai tries to use Ollama on a non-standard port (8080).

I think it should use or at least default to the standard Ollama URL of http://127.0.0.1:11343 - this would allow people to install cai and just start using it without having to customise Ollama or or other compatible tooling that uses Ollama default port 11343.

```
cai local test
ERROR:
error sending request for url (http://localhost:8080/v1/chat/completions)

ollama serve
listen tcp 127.0.0.1:11434
```

**Comment by sammcj** [comment]
PR submitted #5

**Comment by ad-si** [comment]
Thanks for the heads up and the PR!
I implemented it a little differently, but I hope you like it? -> a3aac656d72df55ad37424dfde24aa0a9daa3607

TLDR:
```sh
cargo install cai
cai ollama llama2 How heigh is the Eiffel Tower in meters
```

**Comment by sammcj** [comment]
That works! Thanks a bunch :) 

```
$ cai ollama tinydolphin:1.1b-v2.8-q5_K_M 'tell me a joke'

🧠 Model: Ollama's tinydolphin:1.1b-v2.8-q5_K_M
⏱️ Duration: 706 ms


 Why is there a hole in the ground? Because if you drop a penny and it goes
through, it'll take us to the moon.
```
